WORKSPACE

```diff
@@ -106,6 +90,24 @@ load("@io_grpc_grpc_java//:repositories.bzl", "IO_GRPC_GRPC_JAVA_ARTIFACTS", "IO
 
 grpc_java_repositories()
 
+# Google Protocol Buffers.
+# https://github.com/protocolbuffers/protobuf
+#
+# Protobuf should be loaded later than C++ gRPC in the WORKSPACE file
```

Comment:
```suggestion
# Protobuf should be loaded after `grpc_deps` in the WORKSPACE file
```

(if that's correct)

Reply:
It doesn't have to be immediately after

Reply:
I didn't say "immediately after" :) "later than" does not seem a common way to refer to the order of lines in a file. Anyways up to you, I think it's understandable. But please refer to the actual function you are referring to, for clarity.

Reply:
I don't see it -- did you forget to push?

Reply:
Yes, I meant to click start a review, but responded immediately